configure.ac

```diff
@@ -98,7 +98,6 @@ AC_CONFIG_FILES([
 	contrib/pyzfs/setup.py
 	contrib/zcp/Makefile
 	contrib/zstd/Makefile
-	contrib/zstd/Makefile
```

Comment:
This should be backported to an earlier commit!

Reply:
yeah, I plan to merge back multiple changes. I just wanted to wait until all tests run smooth :)

Reply:
(guess it gets squashed together anyway when this is stable, but just for info)